great_expectations/util.py

```diff
@@ -1137,6 +1137,24 @@ def is_float(value: Any) -> bool:
     return True
 
 
+def is_nan(value: Any) -> bool:
+    """
+    If value is an array, test element-wise for NaN and return result as a boolean array.
+    If value is a scalar, return boolean.
+    Args:
+        value: The value to test
+
+    Returns:
+        The results of the test
+    """
+    import numpy as np
+
+    try:
+        return np.isnan(value)
+    except TypeError:
+        return True
+
+
```

Comment:
Could we write some basic unit tests for this helper? Maybe one for each path?

Reply:
@cdkini Probably overkill -- for `isnan`.